README.md

```diff
@@ -495,6 +495,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Anurag Sharma](https://github.com/Anurag30112003)
   - [Anurag Singh](https://github.com/AnuragProg)
   - [Anusha Chaturvedi](https://github.com/anusha-c18)
+  - [Anushadevi Rajkumar](https://github.com/AnushaDeviR)
   - [Anusha Hiremath](https://github.com/AnushaHiremath)
   - [Anusha S](https://github.com/anushas-dev)
```

Comment:
```suggestion
  - [Anusha Hiremath](https://github.com/AnushaHiremath)
  - [Anusha S](https://github.com/anushas-dev)
  - [Anushadevi Rajkumar](https://github.com/AnushaDeviR)
```

Reply:
Apologies for the misunderstanding, thanks for stating it out. 🙏🏼